src/screens/regimen/SaloonVisitScreen/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import { NavigationInterface } from '../../types';
+import Data from '../../../libs/visit.json';
+import Visits from './visits';
+import {
+    Container,
+    SubTitle,
+    Title,
+    Content
+} from './style';
+
+interface RoutineProps extends NavigationInterface {
+    testId?: string;
+}
+
+export default function SaloonVisitScreen(props: RoutineProps) {
+    return (
+        <ScrollView showsVerticalScrollIndicator={false} style={{ flex: 1 }}>
```

Comment:
Please always use FlatList because this content will eventually become large over the years, which could cause performance issue.